swap_meet/decor.py

```diff
@@ -1,2 +1,11 @@
-class Decor:
-    pass
\ No newline at end of file
+
+from .item import Item
+class Decor(Item):
+
+    def __init__(self, id = None, condition = None, width = None, length = None):
+        super(). __init__(id, condition)
+        self.width = 0 if width == None else width
+        self.length = 0 if length == None else length
+
+    def __str__(self):
```

Comment:
Check out my comment in clothing.py about how to use the parent class method in this method.